app/views/migration/_form.html.erb

```diff
@@ -42,6 +42,18 @@
           <% end %>
         <% end %>
       <% end %>
+
+      <%= agg.nested_fields_for(:edm_hasViews, legend: i18n.t('legends.each_object', number: '')) do |wr| %>
```

Comment:
Additional objects need their number included in the legend. At present all just display "Object".

Reply:
nested_form_fields provides a method to access indexes via __nested_field_for_replace_with_index__ however the numbers for the labels don't quite add up since it's 0 indexed plus we have an additional object as edm_isShownBy. So @andyjmaclean added some custom logic to the styleguide to re label the individual sections when one is added/removed or the page is reloaded.